server/server/Cargo.toml

```diff
@@ -51,6 +51,7 @@ futures = "0.3"
 fast_log = { version = "1.5" }
 rcgen = "0.9.2"
 
+
```

Comment:
Nit: extra line here...

Reply:
oops, saw this too late. have removed, will pop in the next PR